game-cards/SGgame.js

```diff
@@ -382,13 +382,20 @@ exports.commands = {
 			type = "Fire";
 		} else if (starters.indexOf(target) >= 14) {
 			type = "Water";
-		}
+		} else if (starters.indexOF(target) = 15) {
```

Comment:
`=` is for assignment. ex: `let hello = 'world';`

use `===` for comparing. ex: `starters.indexOF(target) === 15`